doc/getting_started/setup_verilator.md

```diff
@@ -21,16 +55,13 @@ First the simulation needs to built itself.
 $ cd $REPO_TOP
 $ fusesoc --cores-root . run --flag=fileset_top --target=sim --setup --build lowrisc:dv:chip_verilator_sim
 ```
-The fsel_top flag used above is specific to the OpenTitan project to select the correct fileset.
-
 
 Then we need to build software to run on the simulated system.
 There are 3 memory types: ROM, RAM and Flash.
```

Comment:
Technically there are 4 memories: ROM, SRAM, Flash, and OTP. Images are only built for ROM, Flash, and OTP. SRAM gets populated at runtime.